src/components/pages/Randomizer/BasicNemesisCards/index.tsx

```diff
@@ -55,40 +54,34 @@ const BasicNemesisCards = ({
     setPlayerCount(parseInt(event.currentTarget.value) as PlayerCount)
   }
 
+  const noCardsRolled =
+    randomBasicNemesisCards.Tier1.length < 1 ||
+    randomBasicNemesisCards.Tier2.length < 1 ||
+    randomBasicNemesisCards.Tier3.length < 1
+
   return (
     <React.Fragment>
       <PlayerAmountPicker
         selectedValue={playerCount.toString()}
         handleAmountChange={handleAmountChange}
       />
-      {randomBasicNemesisCards.Tier1 && (
-        <React.Fragment>
-          <SectionHeadline>Tier 1</SectionHeadline>
-          <BasicNemesisCardList
-            basicNemesisCards={randomBasicNemesisCards.Tier1 || []}
+      {noCardsRolled ? (
+        <EmptyHint>Tab button to draw new basic nemesis cards!</EmptyHint>
+      ) : (
+        <>
```

Comment:
Are we using the short-hand form of React.Fragment now? Personally I like the longer form.

Reply:
Actually i started perferring the shorthand, because it is easier to spot. The long fragment name requires me to read it out, to understand the intent.

Reply:
Alright.